README.md

```diff
@@ -352,12 +352,14 @@ A `load_balancer` block supports the following:
 * `port` - (Optional) The port of the listener. Defaults to 443.
 
 * `container_name` - (Required) The name of the container to associate
-with the load balancer as it appears in the container definition (by
-default, `containers.json`).
+with the load balancer as it appears in the container definition
+(by default, `containers.json`). Defaults to the value specified
+in the [Service Discovery](#service_discovery) block.
 
 * `container_port` - (Required) The port on the container to associate
-with the load balancer as it appears in the container definition (by
-default, `containers.json`).
+with the load balancer as it appears in the container definition
+(by default, `containers.json`). Defaults to the value specified
```

Comment:
This is false as the code is written.

Reply:
Directions unclear. What should the change be to reflect reality? Remove `container_port` variable from the `load_balancer` section?